components/brave_rewards/resources/extension/brave_rewards/background/api/locale_api.ts

```diff
@@ -36,6 +36,7 @@ export const getUIMessages = (): Record<string, string> => {
     'for',
     'grants',
     'includeInAuto',
+    'insufficientFunds',
```

Comment:
`insufficientFundsNotification` is missing here

Reply:
Whoops, added.